assets/javascripts/discourse/connectors/group-reports-nav-item/nav-item.js.es6

```diff
@@ -0,0 +1,14 @@
+import { ajax } from "discourse/lib/ajax";
+
+export default {
+  shouldRender(args, component) {
+    let usersGroupIds = component.currentUser.groups.map(g => g.id)
+    if (!usersGroupIds.includes(args.groupId)) return false // User not a part of group
+
+    return true
```

Comment:
be sure to install prettier locally and always run your files through it. I use IDE integration it is magical.